tripFrontend/src/features/auth/components/SubmitButton/SumitButton.tsx

```
@@ -0,0 +1,18 @@
+import { JSX } from 'react';
+
+type Props = {
+	label?: string;
+	onClick?: () => void;
+};
+
+export const SubmitButton = ({ label, onClick }: Props): JSX.Element => {
+	return (
+		<button
+			type="button"
+			onClick={() => onClick && onClick()}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)